### PR TITLE
tests(shorthand): allow non-strict assert

### PR DESCRIPTION
**Pull request by layershifter**
# Why? 

Extracted from #1795.

[`contains`](https://github.com/airbnb/enzyme/blob/master/docs/api/ShallowWrapper/contains.md) is too strict, we should allow non-strict assert in these cases, useless conditions should gone.

### CODE REVIEW

File: test/specs/elements/Input/Input-test.js [hunk 1]
```diff
@@ -63,6 +63,7 @@ describe('Input', () => {
   common.implementsHTMLInputProp(Input, {
     alwaysPresent: true,
     shorthandDefaultProps: { type: 'text' },
+    strictAssert: false,
```
Comment (levithomason) [inline test/specs/elements/Input/Input-test.js]: How about `assertExactMatch` or similar?  It is not clear _what_ is being strictly asserted.

Comment (layershifter) [reply test/specs/elements/Input/Input-test.js]: Yep, I like your suggestion, will change :+1:

**Comment by layershifter** [comment]
@levithomason I've changed an option name :+1:

**Comment by levithomason** [comment]
Released in `semantic-ui-react@0.71.2`

**Pull request merged**